code/modules/materials/Mat_Fabrication.dm

```diff
@@ -317,7 +317,8 @@
 					var/howMany = input(usr, "How many ([maxAmt] max)?", "Select amount", maxAmt)
 					if(howMany > maxAmt || !selectedRecipe) return //ZeWaka: Fix for null.canBuild
 					if(selectedRecipe.canBuild(howMany, src))
-						selectedRecipe.build(howMany, src)
+						SPAWN(2.5 SECONDS)
+							selectedRecipe.build(howMany, src)
```

Comment:
wat

Reply:
It was to give the sprite some time to play, can remove/lower if you think that'd be better